src/test/java/utils/DriverSetUp.java

```diff
@@ -0,0 +1,41 @@
+package utils;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+import java.io.*;
+import java.util.Properties;
+
+public class DriverSetUp {
+    protected static WebDriver driver;
+    private String filePath = "/Users/avoloshchuk/IdeaProjects/com.sharelane/src/test/resources/app.properties";
+    private Properties appProps = new Properties();
+
+    public void loadPropertiesFromFile() {
+        try (InputStream input = new FileInputStream(filePath)) {
+            appProps.load(input);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @BeforeSuite
+    public void setChromeDriver() {
+        loadPropertiesFromFile();
+        System.setProperty(appProps.getProperty("chromeDriver"), appProps.getProperty("chromeDriverPath"));
+        driver = new ChromeDriver();
+    }
+
+    @BeforeClass
+    public void openBrowser() {
```

Comment:
I think it should be named more descriptive... maybe smith like openMainPage

Reply:
Method name was changed from openBrowser() to loadMainPage()